schedulers/emailCreationScheduler.js

```diff
@@ -24,10 +25,26 @@ const createEmailAndMarrainage = async (user, creator) => {
   }
 };
 
+const differenceGithubOVH = function differenceGithubOVH(user, ovhAccountName) {
+  return user.id === ovhAccountName;
+};
+
+// get the difference between github users and ovh users
```

Comment:
Le terme `difference` ne m'évoque pas vraiment ce que fait la fonction alors que le titre `getUnregisteredOVHUsers` par ailleurs est très parlant. Un texte plus explicite style `get user that have github acount and no email registered on ovh` serait plus parlant pour moi.

Reply:
merci du retour, je modifie :slightly_smiling_face: